### PR TITLE
Math magic: setup

### PR DESCRIPTION
**Pull request by Emmy-github-webdev**
# Math magic: setup

In this project I 

- Create react app
- Ensure there is no linter error
- Follow Gitflow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,63 @@
...
+
+- Major languages: ReactJS,
+- Technologies used: Web services, babelrc
+
+
+# Live Demo
+
+[Live Demo Link]().
+
+
```
Comment (ArthurGC) [inline README.md]: - [ ] You want to consider removing broken links in your README if you don't have a link for it yet. ♻️

**Review by ArthurGC — CHANGES_REQUESTED** [review]
# Required Changes ♻️ :warning: 
Hi  @Emmy-github-webdev 

Amazing job!
There are some issues that you still need to work on to go to the next project but you are almost there! 



## Good Points :+1: :
You used proper GitHub Flow :heavy_check_mark:
Your linter checks are all passing :heavy_check_mark:
Your PR messages are descriptive :heavy_check_mark:
Your commit messages are meaningful :heavy_check_mark:

## Aspects to improve ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Emmy-github-webdev** [comment]
Thank you @ArthurGC. Recommendation implemented

**Review by er-contreras — APPROVED** [review]
Hi @Emmy-github-webdev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- Well done!

Felix

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Emmy-github-webdev** [comment]
Thanks @eri8-9

**Pull request merged**